kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/test/java/org/kie/server/services/jbpm/ui/FormServiceBaseTest.java

```diff
@@ -0,0 +1,140 @@
+package org.kie.server.services.jbpm.ui;
+
+/*
```

Comment:
Please move the licence header declaration before package definition.

Reply:
Done!